content/kubernetes/re-databases/routes.md

```diff
@@ -51,4 +51,11 @@ Use the `openssl` command to access the database from outside the cluster.
   openssl s_client -connect <hostname>:443 -crlf -CAfile ./ca.pem -servername <hostname>
   ```
 
-  Replace the `<hostname>` value with the hostname for your new route.
\ No newline at end of file
+  Replace the `<hostname>` value with the hostname for your new route.
+  
+  To connect via `redis-cli`:
+  
+  ```sh
+  redis-cli -h <hostname> -p 443 --tls --cacert ./ca.pem --sni <hostname>
+  ```
```

Comment:
Maybe place this before the `openssl` example above? I would say the `redis-cli` example is more important and standard way of accessing a Redis DB, whereas the openssl is more of a sanity check used to test the connection.